packages/react/src/components/Table/Table.story.jsx

```diff
@@ -2910,6 +2910,30 @@ WithLoadingState.story = {
   name: 'with loading state',
 };
 
+export const WithCustomTooltip = () => (
```

Comment:
Are these changes meant to be included with this PR. Seem unrelated and adds another story. 😬

Reply:
yeah I needed it to understand how to use Table tooltips :)